# CustomPoint.msg


uint32 offset_time              # Offset time relative to the base time

float32 x                       # X axis coordinate, unit: m

float32 y                       # Y axis coordinate, unit: m

float32 z                       # Z axis coordinate, unit: m

uint8 reflectivity              # Reflectivity, 0~255

uint8 tag                       # Livox tag

uint8 line                      # Laser number in lidar
